Simplenote/src/main/java/com/automattic/simplenote/NoteMarkdownFragment.java

```diff
@@ -129,7 +132,7 @@ public void run() {
                                     }
                                 }
                             },
-                            requireContext().getResources().getInteger(android.R.integer.config_mediumAnimTime)
+                            delay
```

Comment:
From 4ada7bb8735238ad284515f53b6161cdbcd611e2: 

While working on the screenshot automation, the tests started consistently failing after dismissing a note editor activity because the `NoteMarkdownFragment` was not attached to a context from the call edited in this commit.

I'm not sure why that would happen or if its of any concern in production.

With my limited Android knowledge, this seems to be a legit workaround. We access the context at a time in which we are sure it exists. The value we extract is constant over the application lifecycle anyways, so there's no downside in reading it from the context at a different time
than when it's going to be used.

@hichamboushaba @ParaskP7 you folks have worked on this repo recently and might know why that's happening and if there's a better way to do it. It think that `requireContext()` call appeared as part of 16e90824.

Reply:
👋 @mokagio !

Thanks for taking care of this! 🌟 

I think this is a valid change as we need to treat this `delay` as constant and the way you did it has that effect within the `onPageFinished(...)` scope. There are many ways that this can further improve, but I believe this change is enough for now.

🙏

Reply:
Thank you for looking over this 🙇‍♂️